### PR TITLE
Robots should format our code

### PR DESCRIPTION
**Pull request by limonte**
Since we already have `standard` in the devDeps, [prettier-standard-vscode](https://marketplace.visualstudio.com/items?itemName=numso.prettier-standard-vscode) is all we need to automatically prettify the code.

:robot: :robot: :robot:

### CODE REVIEW

**Comment by toverux** [comment]
I'm still a bit skeptical about automatic code reformatters, not because of what they do, but _how_ they do it.

Most have a basic approach: code -> AST -> code. But the _visual layout_ of the code is lost and ignored as soon as the code passes through the lexer. It's too bad, because the output will be sometimes harder to read in some parts of the code, where an human had previously deliberately decided to stay on the same line, or break it here, or chaining like that here, etc.

Some IDEs (like JetBrains' ones) have more intelligent reformatters, where some choices from the developer are kept, and only the code that violates the defined code style will be rewritten.

This is a **personal opinion** only. I'm well aware that strict code style is important in personal and especially public projects. I'm waiting for neuralnet-driven code reformatters to embrace fully that practice because the current state of the art is deceiving to me :smile:

File: src/sweetalert2.js [hunk 1]
```diff
@@ -11,24 +11,36 @@ let queue = []
  * Hopefully to avoid many github issues
  */
 if (typeof Promise === 'undefined') {
-  error('This package requires a Promise library, please include a shim to enable it in this browser (See: https://github.com/limonte/sweetalert2/wiki/Migration-from-SweetAlert-to-SweetAlert2#1-ie-support)')
+  error(
+    'This package requires a Promise library, please include a shim to enable it in this browser (See: https://github.com/limonte/sweetalert2/wiki/Migration-from-SweetAlert-to-SweetAlert2#1-ie-support)'
+  )
```
Comment (toverux) [inline src/sweetalert2.js]: I would have written this:

```
error(
  'This package requires a Promise library, please include a shim to enable it in this browser ' +
  '(see: https://github.com/limonte/sweetalert2/wiki/Migration-from-SweetAlert-to-SweetAlert2#1-ie-support)'
)
```

File: src/sweetalert2.js [hunk 2]
```diff
@@ -11,24 +11,36 @@ let queue = []
...
  */
-const setParameters = (params) => {
+const setParameters = params => {
   // If a custom element is set, determine if it is valid
-  if ((typeof params.target === 'string' && !document.querySelector(params.target)) || (typeof params.target !== 'string' && !params.target.appendChild)) {
+  if (
+    (typeof params.target === 'string' &&
+      !document.querySelector(params.target)) ||
+    (typeof params.target !== 'string' && !params.target.appendChild)
+  ) {
```
Comment (toverux) [inline src/sweetalert2.js]: Hideous :smile: Maybe we should use more variables here!

File: src/sweetalert2.js [hunk 3]
```diff
@@ -11,24 +11,36 @@ let queue = []
...
+  let targetElement = typeof params.target === 'string'
+    ? document.querySelector(params.target)
+    : params.target
   // If the model target has changed, refresh the modal
-  if (oldModal && targetElement && oldModal.parentNode !== targetElement.parentNode) {
+  if (
+    oldModal &&
+    targetElement &&
+    oldModal.parentNode !== targetElement.parentNode
+  ) {
```
Comment (toverux) [inline src/sweetalert2.js]: :/

File: src/sweetalert2.js [hunk 4]
```diff
@@ -41,11 +53,15 @@ const setParameters = (params) => {
...
+  modal.style.width = typeof params.width === 'number'
+    ? params.width + 'px'
+    : params.width
 
   modal.style.padding = params.padding + 'px'
   modal.style.background = params.background
-  const successIconParts = modal.querySelectorAll('[class^=swal2-success-circular-line], .swal2-success-fix')
+  const successIconParts = modal.querySelectorAll(
+    '[class^=swal2-success-circular-line], .swal2-success-fix'
+  )
```
Comment (toverux) [inline src/sweetalert2.js]: 3 lines for a query selector?

File: src/sweetalert2.js [hunk 5]
```diff
@@ -409,7 +446,10 @@ const sweetAlert = (...args) => {
       break
 
     default:
-      error('Unexpected type of argument! Expected "string" or "object", got ' + typeof args[0])
+      error(
+        'Unexpected type of argument! Expected "string" or "object", got ' +
+          typeof args[0]
+      )
```
Comment (toverux) [inline src/sweetalert2.js]: FOUR lines??

File: src/sweetalert2.js [hunk 6]
```diff
@@ -850,7 +919,15 @@ const sweetAlert = (...args) => {
...
-    const inputTypes = ['input', 'file', 'range', 'select', 'radio', 'checkbox', 'textarea']
+    const inputTypes = [
+      'input',
+      'file',
+      'range',
+      'select',
+      'radio',
+      'checkbox',
+      'textarea'
+    ]
```
Comment (toverux) [inline src/sweetalert2.js]: Kill it with fire!

**Comment by toverux** [comment]
(I've added a few comments on the diff so you can see what I mean - not everywhere, just a few examples)

**Comment by toverux** [comment]
I've added a few comments as examples (not for every diff that I didn't liked). It's too bad that code reformatters are a little dumb, because I like a lot of other changes it made.

File: src/sweetalert2.js (hunk 6 above)
Comment (limonte) [reply src/sweetalert2.js]: I'm crying of laugh :DDDDDD

**Comment by limonte** [comment]
I agree with you, let's wait for a smarter prettifier from near future :+1:

**Pull request closed without merging**